### PR TITLE
Removes Obsolete, Unused, and Unmaintained overrides

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Title
Fun fact. Almost every single instance of the word Overridden is misspelled. Thanks a lot. Really made this easy to sort out.

Full Changelist:
- Removes comments on things that actually aren't overridden
- Fixes a few typos on things that are staying overridden
- Removes the Lazarus Injector and Borg Hypo overrides, which were almost identical aside from a different overlay for being Unused (a very, very similar overlay)
- **Removes e-gun overrides, which are entirely unmaintained and poorly executed, meaning any updated sprites are missed entirely. This is why the Captain's E-Gun has been outdated for months.
This one has only one real 'override removal' present, considering all of the other E-Guns were unchanged anyways barring the outdated ones: the primary Energy gun is now TG's standard, which while slightly dated fits in with all other energy guns' styles of our Upstream.**
- Re-overrides the Ion Rifle and leaves the "nuclear rifle" override. Both are very outdated on TG, and ours are passable.
- Removes an override of the PKA sprite that didn't even work and linked to outdated sprites if it even did work.
- **Removes all Lighting icon overrides. Any changes involving actual brightness and function are not touched, only the icons, as TG just resprited them and ours were causing actual, visible in-game icon conflicts, with flickering states or error icons.**
- **Removes the Surgery Table override, as it's almost the exact same as the Bay reference used for TG's first 2 resprite attempts. We should move away from it and to TG's third and approved resprite before any issues arise.**
- Fixes outdated PKA pathing in two instances

## How This Contributes To The Skyrat Roleplay Experience
Making a game look good is pretty important. Maintaining those looks is a VERY important part of it that isn't ever done.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  Lights
  
image

Surgery
image

Hyposprays
image

And guns.
image
How many of these guns do you think were **actually** un-overridden?
The answer: Four. All in a row starting from the top-left.
The first two, as base types, can't be safely overridden and resprites should be taken upstream, or making them NOT base types. 
The Captain one has been updated to TG standard by finally not using an outdated file.
And the fourth one... I've never even seen it, but the change isn't noticable - just a lighter frame of the exact same sprite quality.
</details>

## Changelog

:cl:
image: removed some obsolete Skyrat icon overrides, moving these items to TG standard: Eguns, Laz/Borg hypos, Lights, and Surgery Tables.
fix: PKAs now have no safeties, as intended
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/aesthetics/lights/code/lighting.dm [hunk 1]
```diff
@@ -6,8 +6,6 @@
 	light_power = 1.25
 
 /obj/machinery/light
-	icon = 'modular_skyrat/modules/aesthetics/lights/icons/lighting.dmi'
```
Comment (lessthnthree) [inline modular_skyrat/modules/aesthetics/lights/code/lighting.dm:9]: Any references to the "tube_lpower" overlay in code needs to be changed to "tube_emergency"

**Review by lessthnthree — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/aesthetics/lights/code/lighting.dm (hunk 1 above)
Comment (lessthnthree) [reply modular_skyrat/modules/aesthetics/lights/code/lighting.dm:9]: image

Have you tested other icon states?

Comment (ghost) [reply modular_skyrat/modules/aesthetics/lights/code/lighting.dm:9]: Shiit I seen it, we had an entirely unique state for it that was labelled "Skyrat Edit" with no tie in to Lighting or Aesthetics 💀 I'll go through and test 'em all now

Comment (ghost) [reply modular_skyrat/modules/aesthetics/lights/code/lighting.dm:9]: Tested them all, for both tubes and bulbs, and floorlights. 
On, Broken, Burnt, No Power, Off
image

**Review by lessthnthree — APPROVED** [review]

**Pull request merged**